src/main/java/org/owasp/esapi/util/ObjFactory.java

```diff
@@ -45,10 +45,11 @@
  */
 public class ObjFactory {
 
-	private static final int CACHE_INITIAL_CAPACITY = 4096;
+    private static final int CACHE_INITIAL_CAPACITY = 32;
```

Comment:
Any real motivation for this change?  are these bytes or MB?

Reply:
If I'm reading the docs right, this is just the initial quantity of elements that the Map can hold.  When 'load factor' % of 'initial capacity' is reached, the map will increase in size.  

https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/ConcurrentHashMap.html#ConcurrentHashMap-int-float-

Reply:
@jeremiahjstacey Exactly. And no way we need 4K. There are only 11 calls to `ObjFactory.make()` in ESAPI.java which will be the usual case, and it does automatically grow as needed, so there's little point in going crazy here. Even 32 is probably more than will ever be needed.